docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -67,9 +67,21 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e) -coverprofile cover.out
 
-# Utilize Kind or modify the e2e tests to load the image locally, enabling compatibility with other vendors.
-.PHONY: test-e2e  # Run the e2e tests against a Kind k8s instance that is spun up.
-test-e2e:
+# TODO(user): To use a different vendor for e2e tests, modify the setup under 'tests/e2e'.
+# The default setup assumes Kind is pre-installed and builds/loads the Manager Docker image locally.
+# Prometheus and CertManager are installed by default; skip with:
+# - PROMETHEUS_INSTALL_SKIP=true
+# - CERT_MANAGER_INSTALL_SKIP=true
+.PHONY: test-e2e
+test-e2e: ## Run the e2e tests. Expected an isolated environment using Kind.
+	@command -v kind >/dev/null 2>&1 || { \
+		echo "Kind is not installed. Please install Kind manually."; \
+		exit 1; \
+	}
+	@kind get clusters | grep -q 'kind' || { \
+		echo "No Kind cluster is running. Please start a Kind cluster before running the e2e tests."; \
+		exit 1; \
+	}
```

Comment:
Hi @DerekTBrown, @majewsky

The changes on this PR should be enough to sort out the scenarios that you raised.
See that we need to have kind up and running for the e2e-tests
Therefore, if anyone run the integration tests without customize them against an env which is not using kind then the test will fail.

c/c @TAM360

Reply:
@camilamacedo86 I think the documentation should also include the information about newly introduced environment variables, and the purpose behind them.

Reply:
It is documented in the makefile target (see this file)
Also, in the e2e tests scaffolds golang docs.

Reply:
@camilamacedo86 wouldn't it be better idea to include it in the docs as well?

Reply:
> @camilamacedo86 wouldn't it be better idea to include it in the docs as well?

It is not the goal of this PR
but we can add that under reference in a follow up.
I am totally OK with. Could you please raise an issue to track the need?

Reply:
@camilamacedo86  I'll create a new issue, once this PR merges.